pkg/filter/dt/filter_mysql_prepare.go

```diff
@@ -146,6 +155,8 @@ func (f *_mysqlFilter) processAfterPrepareInsert(ctx context.Context, conn *driv
 func (f *_mysqlFilter) processAfterPrepareUpdate(ctx context.Context, conn *driver.BackendConnection,
 	stmt *proto.Stmt, updateStmt *ast.UpdateStmt) error {
 	has, xid := misc.HasXIDHint(updateStmt.TableHints)
+	newCtx, span := tracing.GetTraceSpan(ctx, "mysql_process_after_prepare_update")
```

Comment:
I recommend not adding TraceContext for this location. instead, add inside of func `executor.AfterImage`:
```
newCtx, span := tracing.GetTraceSpan(ctx, "fetch_after_image")
```